SkietbaanBE/SkietbaanBE/Controllers/UserController.cs

```diff
@@ -56,10 +70,13 @@ public async Task<IActionResult> AddUser([FromBody] User user)
                 await _context.SaveChangesAsync();
 
                 return Ok("User saved successfully");
-            }else{
```

Comment:
Check the C# standards, indentation should be applied as well as if - else statements being on new lines.

Reply:
I fixed the indentation. Please resolve the conversation

Reply:
Sorted